src/audio/codec_adapter/codec/generic.c

```diff
@@ -259,9 +260,9 @@ int codec_process(struct comp_dev *dev)
 	uint32_t codec_id = cd->ca_config.codec_id;
 	struct codec_data *codec = &cd->codec;
 
-	comp_dbg(dev, "codec_process() start");
+	comp_info(dev, "codec_process() start");
 
```

Comment:
comp_info() here would flood the log. Is this intended?

Reply:
Residue from last debug session before push :)
Thanks for spotting, fixed now.